axum-macros/src/attr_parsing.rs

```diff
@@ -62,7 +62,7 @@ where
 {
     attrs
         .iter()
-        .filter(|attr| attr.path.is_ident(ident))
+        .filter(|attr| attr.meta.path().is_ident(ident))
```

Comment:
Having just upgraded another project to syn 2, I realized you used a more verbose solution than necessary here. `Attribute` also has a `path` method, so this can be `attr.path()`.